Orchestrator/README.md

```diff
@@ -1,34 +1,49 @@
 
-[Orchestrator][18] is a [transformer][4] based solution that is optimized for conversational AI applications. It is built ground-up to run locally with your bot.
+<!-- Coining the term "orchestration" doesn't seem necessary. -->
+
+<!-- Discussing the underlying technology also doesn't seem necessary here. Mention in the technical overview, instead. -->
+
+Orchestrator is an LU solution optimized for conversational AI applications. It is built ground-up to run locally with your bot. See the [technical overview][18] for additional details.
 
 ## Scenarios
-**Dispatch**: Orchestrator is a successor to [dispatch][5]. You can use Orchestrator instead of the current dispatch solution to arbitrate across your [LUIS][1] and [QnA Maker][2] applications. With Orchestrator, you are likely to see:
+
+**Routing**: For bots, Orchestrator can replace the [LUIS Dispatch tool][5]. You can use Orchestrator instead of Dispatch to aggregate multiple [LUIS][1] and [QnA Maker][2] applications. With Orchestrator, you are likely to see:
 
 - Improved classification accuracy
 - Higher resilience to data imbalance across your LUIS and QnA Maker authoring data.
 - Ability to correctly dispatch from relatively little authoring data.
 
-**Intent Recognizer**: You can use Orchestrator as an intent recognizer with [Adaptive dialogs][6]. Using the same approach as in the dispatch scenario above in order to route to responses within your bot logic.
+**Intent recognition**: You can use Orchestrator as an intent recognizer with [adaptive dialogs][6] to route user input to an appropriate skill or sub-component.
 
-**Entity Extraction** is not supported yet.  It is on the planning roadmap to add entity extraction in the future.
+**Entity extraction** is not yet supported.
 
-## Authoring Experience
+## Authoring experience
 
 Orchestrator can be used in different development environments:
 
-* **Code First**: Orchestrator can be integrated into your code project by replacing LUIS for intent recognition such as for skill delegation or dispatching to subsequent language understanding services.  See [Runtime Integration](#runtime-integration) section for more.
-* [Bot Framework Composer][19]: Orchestrator can be selected as a recognizer within Bot Framework Composer. At this point there are limitations to using Orchestrator in Composer primarily around importing of existing models and tuning up recognition performance (* currently available only if the feature flag is enabled with Composer).
+- [Bot Framework SDK][24]: Orchestrator can be integrated into your code project by replacing LUIS for intent recognition, such as for skill delegation or dispatching to subsequent language understanding services. See the [SDK integration](#sdk-integration) section for more information. <!--We don't yet document Orchestrator in the SDK docs. Do we need to?-->
```

Comment:
Orchestrator is still PREVIEW. We should plan to doc in MS Docs by GA ~BUILD timeframe. Not now.